src/components/App/App.jsx

```diff
@@ -0,0 +1,57 @@
+import { useState, useEffect } from "react";
+import "./App.css";
+import { list } from "../../data.js";
+import { TodoList } from "../TodoList/TodoList.jsx";
+import { AddTodoForm } from "../AddTodoForm/AddTodoForm.jsx";
+
+const App = () => {
+	const [todoList, setTodoList] = useState([]);
+
+	const [isLoading, setIsLoading] = useState(true);
+
+	const toLocalStorage = () => {
+		if (isLoading === false) {
+			localStorage.setItem("savedTodoList", JSON.stringify(todoList));
+		}
+	};
+
+	useEffect(() => {
+		new Promise((resolve, reject) => {
+			setTimeout(() => {
+				resolve({
+					data: {
+						todoList: JSON.parse(localStorage.getItem("savedTodoList")) || list,
```

Comment:
We should delete this pre-populated list here. I think a user would like it better if their to-do list didn't have stuff they didn't write

Reply:
deleted.